addons/mail/static/src/models/channel.js

```diff
@@ -219,5 +219,9 @@ registerModel({
         unknownMemberCount: attr({
             compute: '_computeUnknownMemberCount',
         }),
+        /**
+         * Useful to compute the invitation link.
+         */
```

Comment:
Bad comment. Too prone to become a lie in the future, which would make existence of this comment more harmful than if we had no comment at all. And also a good comment on a field should tell the conceptual "what", not just a "why" that gives an example of usage at the time you write the comment but could be just a lie in the future.

Would just drop comment if you don't find a proper better comment.